front/src/components/Common/Header/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import {NavLink} from 'react-router-dom'
+import './header.scss'
+
+const Header = () => {
+    return(
+        <nav>
+            <p>
+                VELVET SCREEN
+            </p>
+            <ul>
+                <li><NavLink to="./Home" activeClassName="selected">Главная</NavLink></li>  
```

Comment:
Russian language here but english in other places. Please choose one or think about adding localization. We add localization in any case in future